src/backend/web/authentication.js

```diff
@@ -82,8 +82,16 @@ function init(passport) {
 
   // Add session user object de/serialize functions
   passport.serializeUser(function(user, done) {
-    // TODO: is it necessary to use JSON vs. Object?
-    done(null, JSON.stringify(user));
+    const user2 = {
+      name: user['http://schemas.microsoft.com/identity/claims/displayname']
```

Comment:
According to https://github.com/kristophjunge/docker-test-saml-idp#usage, we'll get a `username`, `name`, and `email` from the local php testing IdP, and from Seneca we get `http://schemas.microsoft.com/identity/claims/displayname`.  Let's expect one or the other of these two.  If we don't get one of these, that's an error worth logging vs. using a default.